src/Core.Assets/src/index.ts

```diff
@@ -50,6 +50,17 @@ body:has(.prevent-scroll) {
     --presence-unknown: #d13438;
     --highlight-bg: #fff3cd;
 }
+
+
+[role='checkbox'].invalid::part(control),
```

Comment:
Why did you added this code? This has nothing to do with `InputBase` inheritance.

Reply:
This is needed for validation of all inputBase components. Earlier source location was not correct and code was not complete